transports/quartz/src/test/java/org/mule/transport/quartz/QuartzMultipleConnectorsDifferentConfigTestCase.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.transport.quartz;
+
+import org.junit.Test;
+
+import org.mule.tck.junit4.FunctionalTestCase;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.CoreMatchers.equalTo;
+
+/**
+ * The following bug must be avoided:
+ * When more than one quartz connectors are registered,
+ * the quartz scheduler for these connectors have the same name.
+ * As a consequence of this, only the properties for the first
+ * connector are taken into account as the lookup for the quartz
+ * scheduler always returns the same instance (with the same
+ * name)
+ * <p>
+ * see Jira ticket MULE-11159
```

Comment:
We don't add references to jiras on test, as you can check the commit history to find it. If you want to add it anyway, use @see instead